plugins/services/src/js/containers/service-debug/ServiceDebugContainer.js

```diff
@@ -117,7 +118,8 @@ class ServiceDebugContainer extends React.Component {
       } else {
         introText = 'Rejected offer analysis is not currently supported.';
       }
-    } else if (queue == null || queue.declinedOffers.summary == null) {
+    } else if (!DeclinedOffersUtil.shouldDisplayDeclinedOffersWarning(queue)
```

Comment:
#nit We should clean these `if`-`else` and nested `if`s up at some point too. (Again not part of this PR)